### PR TITLE
Add all set-up files

### PR DESCRIPTION
**Pull request by jorgegoco**
I have added all the necessary files for the youtube page clon eskeleton.

### CODE REVIEW

File: youtube-clon.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <div class="icons right">
+      <i class="material-icons">videocam</i>
+      <i class="material-icons">apps</i>
+      <i class="material-icons">notifications</i>
+      <i class="material-icons display-this">account_circle</i>
+    </div>
+  </header>
+  <main>
+    <div class="side-bar">
+      <div class="nav">
```
Comment (jorgegoco) [inline youtube-clon.html:35]: I would better use the `nav` tag as a semantic tag for better performance.

File: youtube-clon.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <button><i class="material-icons">search</i></button>
+      </form>
+      <i class="material-icons mic">mic</i>
+    </div>
+    <div class="icons right">
+      <i class="material-icons">videocam</i>
+      <i class="material-icons">apps</i>
+      <i class="material-icons">notifications</i>
+      <i class="material-icons display-this">account_circle</i>
+    </div>
```
Comment (jorgegoco) [inline youtube-clon.html:31]: I would suggest you use an unordered list instead of several inline elements.

**Review by jorgegoco — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make the required changes.

Submit for a re-review when you're done.

Thanks

File: youtube-clon.html (hunk 2 above)
Comment (ShadyShawkat) [reply youtube-clon.html:31]: Good job sounding suggestive in your required change, you can also use "Please" or "Kindly" to sound more polite.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @jorgegoco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**